src/main/java/com/mixpanel/android/mpmetrics/Tweaks.java

```diff
@@ -58,8 +58,9 @@ public synchronized boolean isNewValue(String tweakName, Object value) {
             MPLog.w(LOGTAG, "Attempt to reference a tweak \"" + tweakName + "\" which has never been defined.");
             return false;
         }
-
         final TweakValue container = mTweakValues.get(tweakName);
+        if (container == null || container.value == null)
+            return value == null;
```

Comment:
this means that we will consider `value` as a new value when it's `null` and the container's value is also `null`.
can we change this line to `value != null`?